src/components/Reports/ReportEdit.tsx

```diff
@@ -67,11 +67,13 @@ export default function ReportEdit(): JSX.Element {
 
   useEffect(() => {
     const getReport = async () => {
-      const result = await ReportService.getReport(reportIdInt);
-      if (result.requestSucceeded && result.report) {
-        setReport(result.report);
-      } else {
-        snackbar.toastError(strings.GENERIC_ERROR, strings.REPORT_COULD_NOT_OPEN);
+      if (reportIdInt) {
```

Comment:
I suppose reportIdInt will always be > 0 but we should maybe check for reportIdInt !== undefined or !isNaN(reportIdInt) in all these changes (since that's what we were trying to fix)?

Reply:
Yes, we.need to check !== undefined now, but I thought _if (reportIdInt) {_ would be ok

Reply:
I see similar code elsewhere, maybe it's ok.. or just check for !isNaN (instead of undefined)